lib/chef/provider/execute.rb

```diff
@@ -69,6 +69,14 @@ def sensitive?
         !!new_resource.sensitive
       end
 
+      def stream_to_formatter?
+        Chef::Config[:always_stream_execute] || run_context.events.formatter?
```

Comment:
Isn't this going to cause the formatter to always receive and thus print stream output, whereas currently we only do so when log level is info or higher?

It seems to me like the model is that log level determines what is sent to the formatters, rather than making decisions about what to print in the formatter based on log level.

```
( Chef::Config[:always_stream_execute] || Chef::Log.info? ) && run_context.events.formatter?
```